### PR TITLE
A BUG was found that we get error while installing the NVD Tool

### PR DESCRIPTION
**Pull request by aravindb26**
Hi team,
Hope all going well

I have found a BUG that we get an error while installing the NVD Tool. It shows the Impact when small developers want to Install It and test your tool. In these cases, they face the issue and leave it without testing. And also a loss for Facebookincubator

Thank You :)

### CODE REVIEW

**Review by ImanolBarba — CHANGES_REQUESTED** [review]
Hi, thanks for raising this issue.

As of go 1.18, `go get` will in fact not install the binaries

Could you please add a line indicating that this command is for users of go 1.17 and above?

Thanks

**Review by aravindb26 — COMMENTED** [review]
Added the line

**Comment by aravindb26** [comment]
> this command is for users of go 1.17 and above

Hi, @ImanolBarba added the line as you said :)

**Comment by aravindb26** [comment]
@ImanolBarba Have you looked that I have made changes as you asked?

**Comment by ImanolBarba** [comment]
I would prefer if you left the original line, then *added* the go 1.17 line below with the comment prefacing it.

Thanks!

**Review by ImanolBarba — CHANGES_REQUESTED** [review]
As mentioned, preserve the original line please

**Comment by aravindb26** [comment]
> 

Which line i should add? Can you screenshot it? I didn't understood what you want

**Review by aravindb26 — COMMENTED** [review]
Is it good now @ImanolBarba

File: README.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ You need a properly setup Go environment.
 
 ```bash
 go get github.com/facebookincubator/nvdtools/...
+go install github.com/facebookincubator/nvdtools/...@latest
+## This command is for users of go 1.17 and above
```
Comment (ImanolBarba) [inline README.md:46]: This should be before the go install line

**Review by ImanolBarba — CHANGES_REQUESTED** [review]
See comments

**Comment by aravindb26** [comment]
> See comments

Yeah check it, already done that

**Comment by ImanolBarba** [comment]
I don't see the changes I suggested applied, it should read like:

```bash
go get github.com/facebookincubator/nvdtools/...

## This command is for users of go 1.17 and above
go install github.com/facebookincubator/nvdtools/...@latest
```

**Review by aravindb26 — COMMENTED** [review]
Already changed

**Comment by ImanolBarba** [comment]
Changes are still not applied, I suggested you format the commands differently because the way it is now it's not clear how users of one version should run the commands from another version.

We'll update the README.

**Pull request closed without merging**